dev_tools/rubocop/rspec-base.yml

```diff
@@ -0,0 +1,11 @@
+require:
+  - rubocop-rspec
+
+# RSpec
```

Comment:
I originally added this comment to break the alphabetical ordering in `infrastructure.yml` but it's probably not needed anymore now that it's in a separate file.